.husky/reuse

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if command -v reuse &> /dev/null
+then
+    (cd databrowser && reuse lint)
+elif command -v pipx &> /dev/null
+then
+    (cd databrowser && pipx run reuse lint)
+else
+    echo -e "\e[33mWarning: neither reuse nor pipx is installed, omitting reuse compliance check\e[0m"
```

Comment:
I'd add a pointer / link to our reuse documentation. The idea is, that the warning message contains enough information on how to proceed to fix this "problem".